tests/test_port_ssl.py

```diff
@@ -20,7 +20,7 @@ class PortSslSetup(TestCase):
     def test_default_port(self) -> None:
         """verify default port value and that TLS is not used"""
         host = "127.0.0.1"
-        port = 1883
+        expected_port = 1883
```

Comment:
@justmobilize  Why is this variable changed?

Reply:
@brentru it wasn't being passed in to anything and just used below as `expected_port = port` which I than took out.

Reply:
Okay, makes sense.